helloworld/helloworldTests/helloworldTests.swift

```diff
@@ -13,26 +13,17 @@ class HelloworldTests: XCTestCase {
     
     override func setUp() {
         super.setUp()
-        // Put setup code here. This method is called before the invocation of each test method in the class.
     }
     
     override func tearDown() {
-        // Put teardown code here. This method is called after the invocation of each test method in the class.
         super.tearDown()
     }
     
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/marianoabdala/HelloWorld/blob/22929daac6424ddce1b82dc0f5f5b5822ee363b7/helloworld/helloworldTests/helloworldTests.swift#L21"></span>Lines should not have trailing whitespace.</td>
    </tr>
  </tbody>
</table>